frontend/packages/console-shared/src/components/status/Status.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react';
+import {
+  HourglassStartIcon,
+  HourglassHalfIcon,
+  SyncIcon,
+  BanIcon,
+  WarningTriangleIcon,
+  UnknownIcon,
+} from '@patternfly/react-icons';
+
+import StatusIconAndText from './StatusIconAndText';
+import ProgressStatus from './ProgressStatus';
+import ErrorStatus from './ErrorStatus';
+import SuccessStatus from './SuccessStatus';
+import InfoStatus from './InfoStatus';
+import { DASH } from '../../constants';
+
+export type StatusProps = {
+  status?: string;
+  title?: string;
+  iconOnly?: boolean;
+};
+
+const Status: React.FC<StatusProps> = ({ status, title, children, iconOnly }) => {
+  const t = title || status;
+  switch (status) {
+    case 'New':
+      return <StatusIconAndText icon={<HourglassStartIcon />} title={t} iconOnly={iconOnly} />;
```

Comment:
Nit: we can represent the commonly occuring props as

```tsx
const statusProps = { title: title || status, iconOnly };
// ...
<StatusIconAndText {...statusProps} icon={<HourglassStartIcon />} />;
```

The `{...statusProps}` spread should come first, so that we can still customize e.g. `title` or `status` if necessary.

Reply:
Nice!